apps/prairielearn/elements/pl-number-input/pl-number-input.mustache

```diff
@@ -125,11 +125,9 @@
         });
     });
 </script>
-{{#show_correct}}
 <div class="d-inline pl-number-input-feedback">
```

Comment:
Should remove what's inside this tag as well.

Reply:
This should be removed in [ee51086](https://github.com/PrairieLearn/PrairieLearn/pull/7934/commits/ee51086858ce7c37114e79320ac1f2e5e4ca5c07).